package.json

```diff
@@ -62,6 +58,7 @@
     "chai": "3.5.0",
     "conventional-changelog": "^1.1.0",
     "css-loader": "0.26.1",
+    "del": "^2.2.2",
```

Comment:
Yes. It was reported as missing dependency.

https://github.com/wholesale-design-system/components/blob/61cb3b1423c164fec1018917d1d679a2d031ce70/build/tasks/clean.js